src/pretix/base/models/vouchers.py

```diff
@@ -13,8 +13,10 @@
 from .items import Item, ItemVariation, Quota
 
 
-def _generate_random_code():
+def _generate_random_code(prefix=None):
     charset = list('ABCDEFGHKLMNPQRSTUVWXYZ23456789')
+    if prefix:
+        return prefix + get_random_string(length=settings.ENTROPY['voucher_code'] - len(prefix), allowed_chars=charset)
```

Comment:
Yeah, I wasn't quite sure about that either, and I resolved that you'd probably expect a voucher code of the standard length, if you were just using that method. I'll bow to your wishes, though ;)